jekyll/assets/img/compass/globe.svg

```diff
@@ -1,10 +1,5 @@
-{% if include.size %}
-  {% assign size = include.size %}
-{% else %}
-  {% assign size = 12 %}
-{% endif %}
-
-<svg fill="none" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
+<?xml version="1.0" encoding="utf-8"?>
+<svg fill="none" aria-label="Globe" role="img" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This is just a personal preference feel free to ignore if don't like it but I would put height and width beside each other.
```suggestion
<svg fill="none" aria-label="Globe" role="img" height="18" width="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
```